interface/src/app/gene/searchFew/searchFew.js

```diff
@@ -51,17 +50,21 @@ angular.module('adage.gene.searchFew', [
               } else {
                 $scope.noResults = false;
               }
-
-              return data.results;
             },
 
             // Error function
             function(responseObject, responseHeaders) {
               $scope.errors = 'Gene search is temporarily down';
             }
-          ).$promise.then(function(response) {
-            return response.results;
-          });
+          ).$promise.then(
+            function(response) {
+              var selectedIDs = Object.keys($scope.selectedGenes);
+              var geneResultsList = response.results.filter(function(result) {
+                return selectedIDs.indexOf(result.id.toString()) === -1;
+              });
+              return geneResultsList.slice(0, numResultsToReturn);
+            }
+          );
```

Comment:
Ok. If we have this tracked as an issue I'm totally fine with dealing with it that way. I do think a user would consider this a bug if they encountered it... the distinction between "feature limitation" and "bug" doesn't really matter if you can't do what you expected. 🤓